.changeset/tasty-cougars-dress.md

```diff
@@ -0,0 +1,6 @@
+---
+"@redocly/openapi-core": patch
+"@redocly/cli": patch
+---
+
+Updated `operation-tag-defined` configurable rule.
```

Comment:
what was was updated about the rule and why?

Reply:
The rule didn't check for a `tags` definition at the operation level.  I also added the tests for it.

Reply:
What was the issue you're trying to solve @jeremyfiel? Was the linter failing? Or just not reporting the problem?